### PR TITLE
Add issue templates

### PR DESCRIPTION
**Pull request by ChrisMaddock**
This is something I've wanted to do for a while. This adds issue templates - for anyone unfamiliar, see [here](https://help.github.com/en/articles/about-issue-and-pull-request-templates).

There's two problems I'm trying to solve:

1. We have lots of brilliant, enthusiastic bug reporters, but it often takes a bit of back and forth to get the information we required. That's frustrating for the reporter, and time consuming for the team.

1. I wanted to make it clear for first-timers that NUnit is a non-commercial, community-powered project, and promote a bit of 'help us to help you'.

Target audience is newcomers. I'm aware the wording and suggestions don't cover all options (e.g. getting debug logs if you're not using the console runner) - but my opinion is that this is to help the majority, and advanced users can continue to figure these things out themselves. 😄 

What do people think?

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+---
+
+<!--
+Thanks for filing a bug report with NUnit! NUnit is run entirely by volunteers from the community, so we welcome pull requests and contributions in all areas.
+
+To help the others diagnose and reproduce your issue, please include as many of the following as relevant:
+- The full command line being used
+- Method of installation (e.g. NuGet, zip, msi)
+- Log files (Run the console with `--trace=Debug` to produce these)
```
Comment (mikkelbu) [inline .github/ISSUE_TEMPLATE/bug-report.md]: Should we also mention the possibility to run `in-process` to diagnose problems?

File: .github/ISSUE_TEMPLATE/support-other.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+---
+
+<!--
+Having problems? NUnit is run entirely by volunteers from the community, but we're keen to help!
+
+To help the others diagnose and reproduce your issue, please include as many of the following as relevant:
+- The full command line being used
+- Method of installation (e.g. NuGet, zip, msi)
+- Log files (Run the console with `--trace=Debug` to produce these)
```
Comment (mikkelbu) [inline .github/ISSUE_TEMPLATE/support-other.md]: Same comment as for bugs

File: .github/ISSUE_TEMPLATE/feature-suggestion.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+<!-- 
+Thanks for suggesting a feature for the NUnit Console/Engine. NUnit is run entirely by volunteers from the community, so we welcome pull requests and contributions in all areas.
+
+Please describe what you're proposing, and let us know if you'd be willing to submit a pull request as well. It's best to wait for a response from the team before starting to write any code - we want to be sure any new features fit in with the overall vision for NUnit, and aren't going to cause any breaking changes.
```
Comment (mikkelbu) [inline .github/ISSUE_TEMPLATE/feature-suggestion.md]: Nitpick. I don't think that there should be a comma in `"we want to be sure any new features fit in with the overall vision for NUnit, and aren't going to cause any breaking changes"` as there is no subject in the clause after the comma. But the resulting sentence becomes a bit long, perhaps we can rephrase it?

Comment (CharliePoole) [reply .github/ISSUE_TEMPLATE/feature-suggestion.md]: Break sentence after "code"? Agree about the comma.

**Comment by ChrisMaddock** [comment]
Thanks both!

**Pull request merged**